app/views/container_topology/show.html.haml

```diff
@@ -24,56 +25,56 @@
   .legend
     %label#selected
     %div{'ng-if' => "kinds"}
-      %kubernetes-topology-icon{:kind => "ContainerReplicator", "tooltip-placement" => "bottom", "tooltip" => "{{legendTooltip}}"}
+      %kubernetes-topology-icon{tooltipOptions, :kind => "ContainerReplicator"}
```

Comment:
> This should be **tooltipOptions, otherwise, you're not creating a valid hash. (Yes, it seems to work, but I can't actually find any docs that would mention it should.)

@himdel I saw couple of examples in various forums without the 2 star chars, and it seemed to work so I went along with it. What those 2 stars are? (they look like pointers :))